static/app/components/events/interfaces/frame/line.tsx

```diff
@@ -19,7 +19,6 @@ import DebugImage from '../debugMeta/debugImage';
 import {combineStatus} from '../debugMeta/utils';
```

Comment:
Is this still in use?  Not sure if we should maintain feature parity

Reply:
lol